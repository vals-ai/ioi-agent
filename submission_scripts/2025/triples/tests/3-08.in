version https://git-lfs.github.com/spec/v1
oid sha256:225fbbb65c80e0219a34b30b975372ae0ce6fe87e4364d680103c92a40ff81ba
size 6040
