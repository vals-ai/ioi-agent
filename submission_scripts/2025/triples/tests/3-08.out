version https://git-lfs.github.com/spec/v1
oid sha256:5edddabca6980abfb97404195059eea0b5a83cea91f7025dc2054ea0ef0be0a9
size 39
